w3-account.md

```diff
@@ -25,7 +25,9 @@ In w3 family of protocols, a namespace is identified by a [`did:key`] identifier
 1. Synchronizing delegations to namespaces across multiple user agents on multiple devices is difficult because of the use of non-memorable [`did:key`] identifiers.
 2. Recovering access if the user loses access to all devices is also a challenge.
 
-To address these issues, we propose the concept of an account as a way to aggregate and manage capabilities under a human-meaningful identifier such as an email address. Specifically, we propose deriving an account identifier from a user-controlled email address in the form of a [`did:mailto`] identifier, which can act as the [principal] in [UCAN] delegation chains.
+To address these issues, we propose the concept of an account as a way to aggregate and manage capabilities under a human-meaningful identifier such as an email address.
+
+Specifically, with did:mailto Accounts we propose deriving an account identifier from a user-controlled email address in the form of a [`did:mailto`] identifier, which can act as the [principal] in [UCAN] delegation chains.
```

Comment:
I think the previous version was clearer. I read it as this document is proposing an account that is identified by an email address, in the form of a `did:mailto` DID.

In your change you've added `did:mailto` _before_ it is introduced...

```suggestion
Specifically, we propose deriving an account identifier from a user-controlled email address in the form of a [`did:mailto`] identifier, which can act as the [principal] in [UCAN] delegation chains.
```

Reply:
@alanshaw I'm trying to introduce a distinction between the two ideas:
* " we propose the concept of an account as a way to aggregate and manage capabilities under a human-meaningful identifier such as an email address."
* and the more specific idea of using an email address as the identifier

i.e. was thinking the former could be 'accounts' and the latter more specifically 'did:mailto accounts'  (accounts that uses did:mailto as opposed to another identifier).

`did:mailto` is somewhat useful for specific use cases, but I'd love to be able to explain that you can use other DIDs too and not just that one.

Just explaining my rationale. It's not super important for me to update this copy right now, and not obvious how to do so for the above messaging goals. So i'm going to accept your suggestion here. (I'm also happy to un-modify this section alltogether since it is irrelevant to addign implementations, my bad)